src/main/java/seedu/address/model/task/Description.java

```diff
@@ -8,6 +8,16 @@
  */
 public class Description {
 
+    public static final String MESSAGE_CONSTRAINTS =
+            "Descriptions should not be blank or starting with white spaces," +
```

Comment:
Minor thing but should this be  `"Description should not be blank or start with whitespaces,"`

Should `Descriptions` be `Description` instead? Since there is only one `Description` in each task, and `start` instead of `starting` to remain consistent with the prior part about how the description `should not be blank`!